### PR TITLE
Moves oAuthClientId and oAuthAppId into private.yml and adds guidance…

### PR DESCRIPTION
**Pull request by gwsii**
… for provisioning Login with Amazon account.

### CODE REVIEW

File: web/serverless.yml [hunk 1]
```diff
@@ -14,8 +14,8 @@ custom:
         - Ref: HelloRetailWebBucket
         - '/*'
 
-  oAuthClientId: amzn1.application-oa2-client.d38cc5e49f9343b5b7a718f9d21fc93a
-  oAuthAppId: amzn1.application.0bfd7ce688a440a1ad215923053e1ec6
+  oAuthClientId: ${self:custom.private.loginWithAmazonClientId}
```
Comment (erikerikson) [inline web/serverless.yml]: 👍

File: README.md [hunk 2]
```diff
@@ -36,6 +36,16 @@ If you are responsible for deploying this system, you'll want to do the followin
     <stage>: +12345678901 # full `+<countryCode>` and 10 digit number
   ```
 
+5. Create an [Login with Amazon](http://login.amazon.com/) account to add authentication and identity using oAuth.
+
+6. In the *Login with Amazon Developer Center* -> Sign into the *App Console* -> *Register new application* button. Fill out requested information about the application.
+
+7. Copy the _Application ID_ to the `private.yml` in the property `loginWithAmazonAppId` and the _Client ID_ to the property `loginWithAmazonClientId`.
```
Comment (erikerikson) [inline README.md]: copy-and-paste-able example?  (as in 4 above)

File: README.md [hunk 3]
```diff
@@ -36,6 +36,16 @@ If you are responsible for deploying this system, you'll want to do the followin
...
 
+5. Create an [Login with Amazon](http://login.amazon.com/) account to add authentication and identity using oAuth.
+
+6. In the *Login with Amazon Developer Center* -> Sign into the *App Console* -> *Register new application* button. Fill out requested information about the application.
+
+7. Copy the _Application ID_ to the `private.yml` in the property `loginWithAmazonAppId` and the _Client ID_ to the property `loginWithAmazonClientId`. 
+
+8. Expand *Web Settings* and click the *Edit* button.
+
+9. In the *Allowed JavaScript Origins* enter the Fully Qualified Domain Name for your hosted websites, e.g. `https://hello-retail.biz`. Local development requires that `https://localhost:7700` is allowed as an origin. This application does not employ return URLs. Once the list of origins is complete, click "Save".
```
Comment (erikerikson) [inline README.md]: Perhaps
e.g. `https://hello-retail.biz` *and `https://<stage>.hello-retail.biz`.

Also, maybe `hello-retail.biz` should be replaced with `<host>`?

File: README.md (hunk 2 above)
Comment (gwsii) [inline README.md]: Good idea. Included that.

File: README.md (hunk 3 above)
Comment (gwsii) [inline README.md]: Nice point. I've revised that.

**Pull request merged**